lib/src/fileset.rs

```diff
@@ -159,6 +161,47 @@ impl FilePattern {
         Self::file_glob_at(dir, pattern, true)
     }
 
+    /// Pattern that matches files with the given extension recursively from
+    /// cwd.
+    pub fn cwd_extension_glob(
```

Comment:
nit: rename to `cwd_extension()` because it doesn't support glob patterns in `extension`.